ruby/team_kattbjorn/rubocop.yml

```
@@ -0,0 +1,21 @@
+# This is Team Kattbjörns rubocop configuration, which is based on the general styleguide with a few overrides.
+
+inherit_from:
+  - https://raw.githubusercontent.com/apoex/styleguide/master/ruby/rubocop.yml
+
+# === OVERRIDE ===
+#
+# Why? Use space within hashes just as in blocks.
+#
+Layout/SpaceInsideHashLiteralBraces:
+  Enabled: true
+  EnforcedStyle: space
```

```suggestion
  EnforcedStyle: no_space
```
är det bästa som har hänt mig :D man ser tydligt skillnad på hashar och block 😌

---

Intressant tanke. Jag tycker mest jag stör mig på avsaknaden av spaces, men det är kanske mest en vanesak.

---

ja så var det för mig först också, men nu älskar jag det :D